pkg/greenlake/get.go

```diff
@@ -0,0 +1,67 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP
+
+package greenlake
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/HewlettPackard/hpecli/pkg/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	getPath       string
+	getJSONResult bool
+)
+
+const users = "users"
+
+func init() {
+	glGetCmd.Flags().StringVar(&getPath, "path", "p", "path to a RedFish item")
+	glGetCmd.Flags().BoolVar(&getJSONResult, "json", false, "display result in json")
```

Comment:
We need to decide what our default output format is, and if we will support other formats.
I have been assuming that our default output would be json